.github/workflows/sbt-node-snyk.yml

```diff
@@ -0,0 +1,37 @@
+name: SBT Node Snyk
+
+on:
+  workflow_call:
+    inputs:
+      DEBUG:
+        type: string
+        required: false
+    secrets:
+      SNYK_TOKEN:
+        required: true
+
+jobs:
+    security:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout branch
+              uses: actions/checkout@v2
+
+            - name: Setup debug var
+              run: echo INPUT_DEBUG=${{ inputs.DEBUG }} >> $GITHUB_ENV
+              shell: bash
+
+            - uses: snyk/actions/setup@0.3.0
+            - uses: actions/setup-node@v2
+              with:
+                  node-version-file: '.nvmrc'
+
+            - uses: actions/setup-java@v2
+              with:
+                  java-version: "11"
+                  distribution: "adopt"
+
+            - name: Snyk monitor
+              run: snyk monitor --all-projects ${INPUT_DEBUG:+ -d}
```

Comment:
I haven't used `--all-projects` before, I wonder whether it'll be a bit prescriptive for more complex projects that might want to be specific about which manifests to monitor.

Relatedly, there's potentially other CLI flags we'd want to consider supporting like `--org` `--target-reference` etc.

Reply:
`all-projects` has been an interesting one. As you mention it's not applicable in all cases such as [flexible-content](https://github.com/guardian/flexible-content/blob/main/.github/workflows/snyk.yml). My aim was to make a simple and generic case which would be useable across most projects. In cases where they differ, a manual workflow could be created as an alternative. My assumption is, you are trying to conceive of a way to cover the cases this wouldn't be able to cope with? If so, I think this is a good idea, but would consider this to still be a sensible starting point.